<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Duplicates</class>
 <widget class="QWidget" name="Duplicates">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Duplicates</string>
  </property>
  <widget class="QListView" name="list1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>171</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnquitar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitar Duplicados</string>
   </property>
  </widget>
  <widget class="QListView" name="list2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>171</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>113</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnadd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>96</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
